### ISSUE TITLE
aarch64 build failure

### ISSUE THREAD

**Issue by lsm5** [containers/libpod#5587]
```
# github.com/containers/libpod/pkg/rootlessport
_build/src/github.com/containers/libpod/pkg/rootlessport/rootlessport_linux.go:120:5: undefined: syscall.Dup2
_build/src/github.com/containers/libpod/pkg/rootlessport/rootlessport_linux.go:121:5: undefined: syscall.Dup2
```

**Comment by lsm5** [comment]
https://koji.fedoraproject.org/koji/taskinfo?taskID=42716422

**Comment by mheon** [comment]
@giuseppe I think this may have been your fix for the rootlesskit port forwarding issue

**Comment by giuseppe** [comment]
PR here: https://github.com/containers/libpod/pull/5589

**Comment by lsm5** [comment]
On Tue, Mar 24, 2020 at 10:19:52AM -0700, OpenShift Merge Robot wrote:
> Closed #5587 via #5589.
> 

built fine https://koji.fedoraproject.org/koji/buildinfo?buildID=1481604

Thanks!

-- 
Lokesh
IRC, GitHub: lsm5
GPG: 0xC7C3A0DD
https://keybase.io/lsm5
